lib/meta-boxes.php

```diff
@@ -39,5 +39,177 @@ function igv_cmb_metaboxes() {
    * Reference: https://github.com/WebDevStudios/CMB2/blob/master/example-functions.php
 	 */
 
+  // PORTRAIT
+  $cmb_portrait = new_cmb2_box( array(
+    'id'            => $prefix . 'portrait_metabox',
+    'title'         => esc_html__( 'Options', 'cmb2' ),
+    'object_types'  => array( 'portrait', ), // Post type
+  ));
+
+  $cmb_portrait->add_field( array(
+  	'name'       => esc_html__( 'Youtube ID', 'cmb2' ),
+  	'desc'       => esc_html__( 'From the video URL. 11 character string after watch?v=', 'cmb2' ),
```

Comment:
I'm wondering if we should be trying to parse youtube IDs from URLs for UX reasons? Obv asking properly is fine and good in many ways, but my exp with real users is they often fuck this up